rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -113,6 +115,8 @@
   private static final int DEFAULT_WORKER_LIVENESS_EXPIRY = 60000; // ms
   private static final int DEFAULT_WORKER_LIVENESS_METRIC_INITIAL_DELAY = 30000; // ms
   private static final int DEFAULT_WORKER_LIVENESS_METRIC_INTERVAL = 30000; // ms
+  private static final String DEFAULT_PRESTO_CLUSTER_MANAGER = "com.qubole.rubix.presto.PrestoClusterManager";
```

Comment:
Are you missing some files ? I dont see the implementation of these classes.

Reply:
PrestoClusterManager and Hadoop2ClusterManager already exist. We are using reflection to create instances of these classes.

Reply:
oops - apologies - I should have read code.